akka-docs/src/main/paradox/typed/reliable-delivery.md

```diff
@@ -28,13 +28,13 @@ To use reliable delivery, add the module to your project:
 Normal @ref:[message delivery reliability](../general/message-delivery-reliability.md) is at-most once delivery, which
 means that messages may be lost. That should be rare, but still possible.
 
-For interactions between some actors that is not acceptable and at-least once delivery or effectively once processing
+For interactions between some actors, that is not acceptable and at-least once delivery or effectively-once processing
 is needed. The tools for reliable delivery described here help with implementing that. It can't be achieved
-automatically under the hood without collaboration from the application because confirming when a message has been
+automatically under the hood without collaboration from the application, because confirming when a message has been
 fully processed is a business level concern. Only ensuring that it was transferred over the network or delivered to 
-the mailbox of the actor would not be enough, since it may crash right after without being processed.
+the mailbox of the actor would not be enough, since the actor may crash right before being able to process the message.
 
-Lost messages are detected, resent and deduplicated as needed. In addition, it also includes flow control for
+Lost messages are detected, resent, and deduplicated as needed. In addition, it also includes flow control for
```

Comment:
The way it is now is correct, adding the `,` before the and is not.

Reply:
I was applying the [Oxford comma rule](https://www.lexico.com/explore/what-is-the-oxford-comma), but I guess it's a matter of taste. Will revert to the original version.